src/loose/statement.js

```diff
@@ -375,7 +375,7 @@ lp.parseImportSpecifierList = function() {
   if (this.tok.type === tt.star) {
     let elt = this.startNode()
     this.next()
-    if (this.eatContextual("as")) elt.local = this.parseIdent()
+    elt.local = this.eatContextual("as") ? this.parseIdent() : this.dummyIdent();
```

Comment:
Just a style fix - please note we're using semicolon-less style. Can you please remove the semicolon and I'll happily merge?